dealroom_firestore_connector/__init__.py

```diff
@@ -132,6 +133,25 @@ def stream(collection_ref, *args, **kwargs):
             return ERROR
 
 
+
+def collection_exists(collection_ref: CollectionReference):
+    """A helper method to check whether a collection exists
+
+    Args:
+        collection_ref (CollectionReference): The reference object to check if exists
+
+    Returns:
+        bool: True if it exists, False otherwise.
+    Examples:
+        >>> db = new_connection(project=FIRESTORE_PROJECT_ID)
+        >>> col_ref = db.collection("MY_COLLECTION")
+        >>> print(fc.collection_exists(col_ref))
+        False
+    """
+    docs = collection_ref.limit(1).get()
```

Comment:
Should this get() be the firestore_connector get()? Just in case it fails so that it's handled in the other function.

Reply:
I also think it should be

Reply:
right, thanks!

Reply:
Done